docs/configuration/settings.rst

```diff
@@ -480,6 +480,28 @@ In order to replace it by another one:
     multiauth.policy.basicauth.use = myproject.authn.BasicAuthPolicy
 
 
+Mohawk request Auth
```

Comment:
What about `Hawk Authentication` as the title?

Reply:
Much better since the hawk implementation (library) might change in the future